src/main/java/in/projecteka/consentmanager/dataflow/model/HealthInformationResponse.java

```diff
@@ -0,0 +1,18 @@
+package in.projecteka.consentmanager.dataflow.model;
+
+import in.projecteka.consentmanager.clients.model.RespError;
+import in.projecteka.consentmanager.link.discovery.model.patient.response.GatewayResponse;
+import lombok.Builder;
+import lombok.Value;
+
+import java.util.UUID;
+
+@Value
+@Builder
+public class HealthInformationResponse {
+    UUID requestId;
+    String timestamp;
+    AcknowledgementResponse hiRequest;
+    GatewayResponse resp;
```

Comment:
you can validate resp to be not null by using @NotNull annotation for resp property. also place @valid in controller method for request body. like this (@Valid @RequestBody HealthInformationResponse healthInformationResponse)

Reply:
done